app/models/mod_warning.rb

```diff
@@ -2,13 +2,20 @@ class ModWarning < ApplicationRecord
   # Warning class name not accepted by Rails, hence this needed
   self.table_name = 'warnings'
 
-  belongs_to :community_user
+  scope :global, -> { where(is_suspension: true, is_global: true) }
+
+  belongs_to :community_user, optional: true
+  belongs_to :user, optional: true
   belongs_to :author, class_name: 'User'
 
   def suspension_active?
     active && is_suspension && !suspension_end.past?
   end
 
+  def global_suspension?
```

Comment:
This method does not appear to be used anywhere (including in other files). Is this being used in a way I can't find? If not, should this be left here for potential future use or be removed as redundant?